common/app/views/fragments/email/signup/emailSignUpNewDesign.scala.html

```diff
@@ -0,0 +1,85 @@
+@import com.gu.identity.model.EmailEmbed
+@(  componentClass: String,
+    listName:String,
+    emailEmbedData: EmailEmbed)(implicit request: RequestHeader)
+
+@import common.LinkTo
+
+
+@formId = @{ componentClass + "-newsletter-embed-form" }
+@dummyInputId = @{ componentClass + "-email-sub-input-name" }
+
+@wrapperClass = @{ "newsletter-embed" + " newsletter-embed--" + componentClass  }
+@formClass = @{ "newsletter-embed__form" + " newsletter-embed__form--" + componentClass }
+
+@imageAltText = @{s"${emailEmbedData.name} profile image"}
+
+
+@form = {
+    <form action="@LinkTo(s"/email/$listName")" method="post" id="@formId" class="@formClass" data-email-form-type="@componentClass" data-email-list-name="@listName">
+        @helper.CSRF.formField
+
+        <div class="newsletter-embed__form-wrapper">
+            <label for="newsletter-embed__input">
+                <span>Enter your email address</span>
+            </label>
+
+            <input
+            class="newsletter-embed__input"
+            type="email"
+            name="email"
+            id="newsletter-embed__input"
+            required />
+            <input
+            class="email-sub__text-input u-h"
+            autocomplete="nope"
```

Comment:
should the value be "off" instead?
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

Reply:
Great catch. This is probably wrong in the old design as well as I think it's a straight port over of the code.